test/Upgrade.t.sol

```diff
@@ -7,8 +7,24 @@ import {MockEngineUpgrade} from "test/utils/mocks/MockEngineUpgrade.sol";
 
 contract UpgradeTest is Bootstrap {
     function setUp() public {
-        vm.rollFork(GOERLI_BLOCK_NUMBER);
-        initializeOptimismGoerli();
+        vm.rollFork(BASE_BLOCK_NUMBER);
+        initializeBase();
+    }
+}
+
+contract StorageLayout is UpgradeTest {
+    function test_nonceBitmap_slot() public {
+        uint256 slot = engineExposed.getNonceBitmapSlot();
```

Comment:
**Praise:** These are thoughtful upgrade tests. Good to use this sanity check as standard - I will copy this.